src/Postmark/Model/Suppressions/PostmarkSuppressionRequestStatus.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Postmark.Model.Suppressions
+{
+    /// <summary>
+    /// Status for the result of applying a Suppression request.
+    /// </summary>
+    public enum PostmarkSuppressionRequestStatus
+    {
+        Suppressed, Failed, Deleted
```

Comment:
Maybe we need a better model for these values. Two of them indicate “success” while one indicates failure. It’s also not clear if this is a temp failure, or a permanent one.

Reply:
I have separated this status into two different enums: one for Suppressions and one for reactivation for clarity purposes.

Regarding whether the failure is a temporary or a permanent one, I have added some documentation to point the user to the `Message` field where he would be able to get this information.